extension.driver.php

```diff
@@ -928,7 +933,28 @@ public function checkFrontendPagePermissions($context) {
 				if($isLoggedIn) {
 					$this->getMemberDriver()->logout();
 				}
-				if($this->getMemberDriver()->login($_POST['fields'])) {
+				$canLogIn = true;
+				/**
+				 * Fired just before a Member tries to log in.
+				 * This delegate is fired just before the login call.
+				 *
+				 * @delegate MembersPreLogin
+				 * @param string $context
+				 *  '/frontend/'
+				 * @param boolean can-logged-in
+				 *  If the current login is valid or not
+				 * @param extensionMember $driver
+				 *  The Member Extension driver
+				 * @param array $errors
+				 *  The error array
+				 */
+				Symphony::ExtensionManager()->notifyMembers('MembersPreLogin', '/frontend/', array(
+					'can-logged-in' => &$canLogIn,
```

Comment:
should this be named just `can-log-in`? `can-logged-in` a little awkward to native english speaker.

Reply:
Er actually, it should `can_log_in` to be consistent with how other variables are named in this extension's delegates.

Reply:
👍

Reply:
Well, I did found why I made it like that: There is `is-logged-in` (you see my copy/paste do you?) So I've made it 'can-log-in'

Reply:
If it is ok, you can just approve the PR and I'll release it. Thanks.